libr/core/cmd_anal.c

```diff
@@ -7432,39 +7433,54 @@ static void cmd_anal_abt(RCore *core, const char *input) {
 		RList *paths = r_core_anal_graph_to (core, addr, n);
 		if (paths) {
 			RAnalBlock *bb;
+			char bb_addr[16];
 			RList *path;
 			RListIter *pathi;
 			RListIter *bbi;
-			bool first_path = true, first_bb = true;
-			if (json) {
-				r_cons_printf ("[");
-			}
+			pj_a (pj);
 			r_list_foreach (paths, pathi, path) {
-				if (json) {
-					if (!first_path) {
-						r_cons_printf (", ");
-					}
+				pj_a (pj);
+				r_list_foreach (path, bbi, bb) {
+					snprintf (bb_addr, sizeof (bb_addr), "0x%08" PFMT64x, bb->addr);
```

Comment:
why? use pj_n (bb-addr);

Reply:
But will that print 8 digits in hex with a 0x in the front?